node/sc/bridge_account_test.go

```diff
@@ -30,8 +30,8 @@ import (
 
 // TestBridgeAccountLockUnlock checks the lock/unlock functionality.
 func TestBridgeAccountLockUnlock(t *testing.T) {
-	tempDir := os.TempDir() + "sc"
-	os.MkdirAll(tempDir, os.ModePerm)
+	tempDir, err := ioutil.TempDir(os.TempDir(), "sc")
```

Comment:
Do you need the string "SC"? Can't you just generate a random dir?

Reply:
Yes. According to the ioutil.TempDir() spec, prefix is required to generate random path.